MIGRATION.md

```diff
@@ -2,6 +2,25 @@
 
 These guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
+## `3.0.1` -> `3.1.0`
+
+- Set the CSS units used in the SDK to be relative (`em`) units. If you previously set the font-size of `.onfido-sdk-ui-Modal-inner`, it is recommended that you remove this `font-size` override.
```

Comment:
Maybe add in the end "This is because we are looking to make the SDK compatible with `em`, but first we need to remove media queries which are not really compatible with that unit.

Reply:
I agree. I'll add that sentence